.github/workflows/delivery-docker.yml

```diff
@@ -0,0 +1,27 @@
+name: delivery / docker
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deliver-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Determine version
+        id: version
+        shell: bash
+        run: |
+          echo "::set-output name=VERSION::$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)"
+      - name: Build and Push Image
+        uses: docker/build-push-action@v1
```

Comment:
Nice. I wonder if lifecycle should switch to using this action too.

Reply:
It seems like it is the official docker image, so I figured it was good to use